Adds enable_metrics flag to Sensor.

francocipollone opened the pull request:
Signed-off-by: Franco Cipollone <franco.c@ekumenlabs.com>

# 🎉 New feature

Related to https://github.com/ignitionrobotics/ign-sensors/pull/146#discussion_r684562497

## Summary
`<enable_metrics>` attribute was added to `Sensor`.


## Checklist
- [x] Signed all commits for DCO
- [x] Added tests
- [ ] Added example and/or tutorial
- [ ] Updated documentation (as needed)
- [ ] Updated migration guide (as needed)
- [x] `codecheck` passed (See [contributing](https://ignitionrobotics.org/docs/all/contributing#contributing-code))
- [x] All tests passed (See [test coverage](https://ignitionrobotics.org/docs/all/contributing#test-coverage))
- [ ] While waiting for a review on your PR, please help review [another open pull request](https://github.com/pulls?q=is%3Aopen+is%3Apr+user%3Aignitionrobotics+repo%3Aosrf%2Fsdformat+archived%3Afalse+) to support the maintainers

**Note to maintainers**: Remember to use **Squash-Merge**

scpeters commented:
I recommend looking at https://github.com/ignitionrobotics/sdformat/pull/429 as an example for updating the specification files in the `sdf/` folder and adding tests

francocipollone commented:
> I recommend looking at #429 as an example for updating the specification files in the `sdf/` folder and adding tests

Thanks, I will check.

francocipollone commented:
Hey @scpeters , some updates:
> updating the specification files in the `sdf/` folder 

I've updated only the 1.7 specification file but I found that in other PRs the specification is updated for several versions.
I didn't find information about which version I should target.

Should I add it to the 1.5 and 1.6 too?

> and adding tests

I've added only unit test for the addition of the new attribute.
I realized that there is no integration test involving the sdf file `test/sensor.sdf` 
Should I create the integration test for `sensor` and test the sensor specification?

chapulina reviewed (approved):
Thanks, @francocipollone , LGTM!

> I didn't find information about which version I should target.

[libSDFormat9 supports the 1.7 spec](http://sdformat.org/tutorials?tut=roadmap&cat=developers&), so I think adding to that is enough.
